Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -671,155 +661,155 @@ def _root_.LinearMap.toQuadraticForm (B : LinearMap.BilinForm R M) : QuadraticFo
     simp only [SMulHomClass.map_smul, LinearMap.smul_apply, smul_eq_mul, mul_assoc]
   exists_companion' := ⟨B + B.flip,
     fun x y => by simp only [map_add, LinearMap.add_apply, LinearMap.flip_apply]; abel⟩
+#align bilin_form.to_quadratic_form LinearMap.toQuadraticForm
 
-variable {B : BilinForm R M}
-
-/-- A bilinear form gives a quadratic form by applying the argument twice. -/
-def toQuadraticForm (B : BilinForm R M) : QuadraticForm R M :=
-  B.toLin.toQuadraticForm
-#align bilin_form.to_quadratic_form BilinForm.toQuadraticForm
+variable {B : M →ₗ[R] M →ₗ[R] R}
 
 @[simp]
-theorem toQuadraticForm_apply (B : BilinForm R M) (x : M) : B.toQuadraticForm x = B x x :=
+theorem toQuadraticForm_apply (B : LinearMap.BilinForm R M) (x : M) : B.toQuadraticForm x = B x x :=
```

Comment:
This diff would go away if you used `open LinearMap (BilinForm)` at the top of the file.